### PR TITLE
Master nmilovanov java system example

### PR DESCRIPTION
**Pull request by nmilovanov**

### CODE REVIEW

File: java/messages/src/main/java/com/nedap/retail/messages/Client.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public void destroy() {
      * System API: list
      */
     public List<SystemListPayload> getSystemList() {
-        final WebResource resource = resource("/system/1.0/list");
+        final WebResource resource = resource("/system/v1/list");
```
Comment (nbozic) [inline java/messages/src/main/java/com/nedap/retail/messages/Client.java]: I saw that we changed endpoint `1.0 -> v1` so this is probably good change but will we have backward compatibility issues, is this method used?

Maybe someone more familiar with this endpoint @jasperkremer @bmjansen ?

File: java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/App.java [hunk 2]
```diff
@@ -84,34 +84,60 @@ public App(final String clientId, final String secret, final String url) throws
...
+        sb.append("\n*** Nedap Retail API examples ***");
+        sb.append("\nChoose example you want to run");
+        sb.append("\n1 : Article API");
+        sb.append("\n2 : EPC API");
+        sb.append("\n3 : Epcis API");
+        sb.append("\n4 : ERP API");
+        sb.append("\n5 : System API");
+        sb.append("\n6 : Workflow API");
+        sb.append("\n0 : Quit");
+        sb.append("\n> ");
```
Comment (stanislavgrujic) [inline java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/App.java]: Please use ```System.getProperties("line.separator");``` and extract it as a constant. 
"\n" is not OS safe.

File: java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/App.java [hunk 3]
```diff
@@ -84,34 +84,60 @@ public App(final String clientId, final String secret, final String url) throws
...
+
+        System.out.println(sb.toString());
```
Comment (stanislavgrujic) [inline java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/App.java]: No need to explicitly call ```toString()```.

File: java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java [hunk 4]
```diff
@@ -36,7 +34,7 @@ private static WorkflowEvent makeWorkflowEvent() {
         workflow.setEventTime(DateTime.now());
         workflow.setLocation(LOCATION);
         workflow.setEpcCount((long) 10);
-        workflow.setMessageIds(Arrays.asList("1", "2", "3", "4", "5", "6", "7", "8", "9", "10"));
+        workflow.setMessageIds(Arrays.asList("abc-123", "def-456", "ghi-789"));
```
Comment (stanislavgrujic) [inline java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/WorkflowExample.java]: You could extract this as a constant.

File: java/messages/src/main/java/com/nedap/retail/messages/Client.java (hunk 1 above)
Comment (jasperkremer) [inline java/messages/src/main/java/com/nedap/retail/messages/Client.java]: We should indeed use v1.

**Pull request merged**